### ISSUE TITLE
Update favicon to new branding

### ISSUE THREAD

**Issue by marnovo** [src-d/landing#341]
Follow up from https://github.com/src-d/landing/issues/314.

The favicon is outdated given the now months-old new branding. 
<img width="252" alt="screen shot 2019-01-10 at 12 25 14 pm" src="https://user-images.githubusercontent.com/15906764/50974874-6106c780-14d3-11e9-9ff8-a2c8fd48cd0c.png">

Update the favicon from the landing to the one available in the [Brand Style Guide media kit](https://docs.google.com/presentation/d/1q290ZrAMola1nSwzYmScOo3gsMgEIXKi2BsWgnMY078/edit#slide=id.g499db0cd7a_0_44) (soon to be also on [src-d/design](https://github.com/src-d/design/)).

**Comment by dpordomingo** [comment]
@ricardobaeta could you please ping us once we have replacements for these three ones?
- `ICO` `16x16` https://raw.githubusercontent.com/src-d/landing/master/static/favicon.ico
- `PNG` `32x32` https://github.com/src-d/landing/blob/master/static/img/logos/favicon.png
- `PNG` `64x64` https://github.com/src-d/landing/blob/master/static/img/logos/favicon96.png

**Comment by marnovo** [comment]
@dpordomingo do we really need all these formats/sizes? A.k.a. don't browsers resize these well from a single png for a few years? If there's good reason to do so, @ricardobaeta can provide them.

**Comment by dpordomingo** [comment]
I have no clear `yes/no` answer.
I'd say that in order to keep the current compatibility, we should replace the existent ones, and then investigate later if we should provide less or even more.
From a quick google search, I found that it can be even more complex than I imagined:
https://css-tricks.com/favicon-quiz

_aside:_
Even being an old format, the `favicon.ico` under the web root directory is an old standard already kept for most of the sites. I saw many 404 log errors in the servers and in Chromecast because browsers use to call this resource automatically whenever you visit a site, regardless it's used any other icon.
